Annotation/constituent_golden_set.json

```diff
@@ -742,7 +858,7 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "how or what he is planning",

```

Comment:
I think it's 2 but I'm not really sure how we should separate them

Reply:
yeh I struggled with this one, we can discuss when you're back